lib/pf/snmptrapd.pm

```diff
@@ -42,7 +44,7 @@ sub receiver {
         redis => _redis_client(),
     });
 #    Delay parsing by two seconds to allow snmp to do it's magic
-    $producer->submit_delayed("pfsnmp_parsing", "pfsnmp_parsing", 2000, [$trapInfo, $variables]);
+    $producer->submit_delayed_hashed($WORKER, $switchIp, "pfsnmp_parsing", "pfsnmp_parsing", 2000, [$trapInfo, $variables]);
```

Comment:
$WORKER is not set.

Reply:
Did you copy the var/conf/snmptrapd.conf?